### PR TITLE
Support Service Instance for emctl

### LINKED ISSUE #72: Support query service instance registered in the registry via emctl
emctl should provide subcommand to query service instance, the output looks like:

```
SERVICE                      TENANT        INSTANCE                                            CANARY           STATUS 
-------------------------------------------------------------------------------------------------------------
vets-service        spring-petclinic   vets-service-79b955b989-22b4q          lv1                     ON_LINE
vets-service        spring-petclinic   vets-service-79b955b989-42b4q                                    ON_LINE
vets-service        spring-petclinic   vets-service-79b955b989-32b4g                                    OUT_OF_SERVICE
```

When users query service instance, they can specify a (or many) condition(s) to filter result, conditions:

1. service name, filter by services' name.
2. tenant, filter by tenant name.
3. canary, filter by canary version.
4. status, filter by service instance status.

### PR DESCRIPTION
**Pull request by xxx7xxxx**
Fix #72 

Example:

```
emctl get serviceinstance
emctl get serviceinstance service-001/id-xxx
emctl delete  serviceinstance service-001/id-xxx
```

And for the filter feature, it needs to be a global design instead of only for serviceinstance.

### CODE REVIEW

**Comment by zhao-kun** [comment]
The implementation isn't completely fulfilling the requirements in #72. We temporarily ignore filter, but the outputs don't make sense to users. The output must contain the information about the instance's status and instance label

**Comment by zhao-kun** [comment]
We needn't support deleting instances as they are automatically maintained by the Easemesh control plane.  When an mesh pod terminated the control plane of the EaseMesh would eliminate its instance information from the registry automatically.

**Comment by xxx7xxxx** [comment]
The output issue fixed.

File: emctl/cmd/client/command/printer/printer.go [hunk 1]
```diff
@@ -18,32 +18,46 @@
...
 	Printer interface {
-		PrintObjects(objects []resource.MeshObject)
+		PrintObjects(objects []meta.MeshObject)
 	}
+
 	printer struct {
 		outputFormat string
 	}
+
+	TableColumn struct {
```
Comment (zhao-kun) [inline emctl/cmd/client/command/printer/printer.go:43]: Need be changes according to complain of the GitHub Actions

File: emctl/cmd/client/resource/loadbalance.go [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	// LoadBalance describes loadbalance resource of the EaseMesh
+	LoadBalance struct {
+		meta.MeshResource `yaml:",inline"`
+		Spec              *v1alpha1.LoadBalance `yaml:"spec" jsonschema:"required"`
+	}
+)
+
+var _ printer.TableObject = &LoadBalance{}
+
+func (l *LoadBalance) Columns() []*printer.TableColumn {
```
Comment (zhao-kun) [inline emctl/cmd/client/resource/loadbalance.go:36]: ditto

File: emctl/cmd/client/command/printer/printer.go (hunk 1 above)
Comment (xxx7xxxx) [reply emctl/cmd/client/command/printer/printer.go:44]: Fixed all

**Review by zhao-kun — APPROVED** [review]

**Pull request merged**